Update Set-WindowsSearchSetting.md (#3673)
MicrosoftDocs/windows-powershell-docs: docset/winserver2022-ps/windowssearch/Set-WindowsSearchSetting.md

PS C:\> Set-WindowsSearchSetting -SearchExperienceSetting "Personalized"
```

This command allows Windows Search to use search history, but not the specific location of the user, to personalize results.

### Example 2: Modify Windows Search settings
```
PS C:\> Set-WindowsSearchSetting -EnableWebResultsSetting $True -SafeSearchSetting "Strict" -SearchExperienceSetting "PersonalizedAndLocation"
```

This command modifies Windows Search settings.
The command enables the use of search history and the specific location of the user by specifying a value of PersonalizedAndLocation for the *SearchExperienceSetting* parameter.
The command enables web results and sets SafeSearch to a value of Strict.

> ### Example 3: Enable Metered Web Search

Reviewer: ```suggestion
### Example 3: Enable metered web search
```

```
PS C:\> Set-WindowsSearchSetting -EnableMeteredWebResultsSetting $True
```

> This command enables metered web searching results only.

Reviewer 2: ```suggestion
This command enables Windows Search to displays web results and suggestions while using metered connections.
```

> This command enables Windows Search to displays web results and suggestions while using metered connections.

Reviewer: ```suggestion
This command enables Windows Search to display web results and suggestions while using metered connections.
```
